project/src/main/gameplay-settings.gd

```diff
@@ -3,8 +3,17 @@ class_name GameplaySettings
 Manages settings which control the gameplay.
 """
 
+signal ghost_piece_changed(value)
+
 # 'true' if a ghost piece should be shown during the puzzle sections.
-var ghost_piece := true
+var ghost_piece := true setget set_ghost_piece
+
+func set_ghost_piece(value: bool) -> void:
```

Comment:
This setter doesn't adhere to our conventions; `set_ghost_piece(new_ghost_piece)`

Reply:
Fixed; `func set_ghost_piece(new_ghost_piece: bool) -> void:`